content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+# Multiple Active Runs in One Process (Reinit Behavior)
+
+**Quick Reference**  
+- **`return_previous`**: Return the existing run; no new run is created.  
+- **`finish_previous`**: Finish (end) the old run, then create a new one.  
```

Comment:
`(end)` is redundant and could be removed. I think the docs use "finish" consistently enough for users to know what it means, and it corresponds directly to the `run.finish()` method.